biz.aQute.tester.junit-platform/src/aQute/tester/bundle/engine/discovery/BundleSelector.java

```diff
@@ -0,0 +1,37 @@
+package aQute.tester.bundle.engine.discovery;
+
+import org.junit.platform.engine.DiscoverySelector;
+import org.osgi.framework.Bundle;
+import org.osgi.framework.VersionRange;
+
+public class BundleSelector implements DiscoverySelector {
+
+	private String			symbolicName;
+	private VersionRange	version;
```

Comment:
These fields should be final which means you need a 2 arg constructor.

Reply:
Done.